pcsx2/x86/microVU_Upper.inl

```diff
@@ -98,15 +99,20 @@ static void mVUupdateFlags(mV, const xmm& reg, const xmm& regT1in = xEmptyReg, c
 		xAND(gprT2, AND_XYZW); // Grab "Is FLT_MAX" bits from the previous calculation
 		xForwardJump32 oJMP(Jcc_Zero);
 			xOR(sReg, 0x820000);
+			if (mFLAG.doFlag)
```

Comment:
Formatting here as well, also the above line.

Reply:
You've moaned about this exact line before, and I told you last time I did that on purpsoe because it was a conditional statement, just like an if statement.

Reply:
Ugh fine, if it'll stop you complaining